src/CommunityToolkit.Maui.Core/Platform/StatusBar/StatusBar.shared.cs

```diff
@@ -18,4 +18,10 @@ public static partial class StatusBar
 	/// <param name="statusBarStyle"> The <see cref="StatusBarStyle"/> that will used by status bar.</param>
 	public static void SetStyle(StatusBarStyle statusBarStyle) =>
 		PlatformSetStyle(statusBarStyle);
+
+	/// <summary>
+	/// Method to update the status bar size.
+	/// </summary>
+	public static void UpdateBarSize() =>
```

Comment:
As this method is only for iOS let's move it to .iOS.cs file

Reply:
@VladislavAntonyuk, Thank you for your review.
Remove it from Shared and move the corresponding method to iOS.